examples/web/examples/meta/index.js

```diff
@@ -54,6 +73,7 @@ function btnAddClick() {
 }
 
 function prepareClickEvents() {
+  // eslint-disable-next-line no-plusplus
```

Comment:
this looks like such a silly rule, it's so common to use ++. Do we really need it? Can we have the config updated to allow it?

Reply:
Sounds good!